src/utilities/effects.ts

```diff
@@ -158,6 +158,24 @@ const effects = {
     }
   },
 
+  async createExpansionSequence(seqId: string, simulationDatasetId: number): Promise<void> {
+    try {
+      creatingExpansionSequence.set(true);
+      const sequence: ExpansionSequenceInsertInput = {
+        metadata: {},
+        seq_id: seqId,
+        simulation_dataset_id: simulationDatasetId,
+      };
+      await reqHasura<SeqId>(gql.CREATE_EXPANSION_SEQUENCE, { sequence });
+      showSuccessToast('Sequence Created Successfully');
```

Comment:
Should we also refer to these as "Expansion Sequence" in the toast message? Or is it mostly for internal clarification.

Reply:
Good question. I went back and forth with this. For now I think 'sequence' still makes sense for the display because that's what it is called in the database. But I think eventually we will change it once we do some DB refactoring.